UW 537 & UW 538: Add docs for MPAS and MPAS Init drivers. (#450)
ufs-community/uwtools: docs/index.rst

Drivers for NWP components are available as top-level CLI modes and API modules.

Provided with a valid UW YAML configuration file, and CLI arguments when required, ``uw`` can prepare a fully provisioned run directory and execute a component either directly on the current system, or via a batch job submitted to an HPC scheduler.

> Each driver produces a list of available ``TASK`` arguments from its CLI ``--help, -h`` flag (shown
> below). The ``provisioned_run_directory`` will do everything except ``run`` the executable, but any
> of the tasks may be requested and only the steps required to produce that task will be performed.

Reviewer: ```suggestion
Each driver produces a list of available ``TASK`` arguments from its CLI ``--help, -h`` flag. The ``provisioned_run_directory`` will do everything except ``run`` the executable, but any of the tasks may be requested and only the steps required to produce that task will be performed.
```
Combine lines, and `(shown below)` made me expect to see an example of the tasks being listed, which isn't actually shown here.

> Over time, we'll add many other drivers to support a variety of UFS components from pre-processing to post-processing, along with many data assimilation components.

Author: This block moved here with slight modifications when I made two subsections here for UFS vs MPAS drivers.

Drivers for UFS
^^^^^^^^^^^^^^^

> Typically, these would be run in the sequence in which they are ordered here, although not all drivers are currently in place to produce all the steps needed to prepare a forecast.

Reviewer: ```suggestion
To prepare a complete forecast, drivers would typically be run in the order shown here (along with additional drivers still in development).
```

sfc_climo_gen
"""""""""""""

| **CLI**: ``uw sfc_climo_gen -h``
| **API**: ``import uwtools.api.drivers.sfc_climo_gen``

chgres_cube
"""""""""""

| **CLI**: ``uw chgres_cube -h``
| **API**: ``import uwtools.api.drivers.chgres_cube``

FV3
"""

| **CLI**: ``uw fv3 -h``
| **API**: ``import uwtools.api.drivers.fv3``

Drivers for MPAS
^^^^^^^^^^^^^^^^

> Drivers for working with standalone :mpas:`Model for Prediction Across Scales (MPAS)<>`. Typically, these would be run in the sequence in which they are ordered here:

Reviewer: ```suggestion
Drivers for working with standalone :mpas:`Model for Prediction Across Scales (MPAS)<>`, typically, run in the order shown here:
```
